src/Emitter.js

```diff
@@ -745,12 +756,39 @@
 	});
 
 	/**
+	 * If particles should be emitted during update() calls. Setting this to false
+	 * stops new particles from being created, but allows existing ones to die out.
+	 * @property {Boolean} emit
+	 */
+	Object.defineProperty(p, "autoUpdate",
+	{
+		get: function() { return this._autoUpdate; },
+		set: function(value)
+		{
+			if (this._autoUpdate && value === false)
+			{
+				ticker.remove(this.update, this);
+			}
+			else if (!this._autoUpdate && value === true)
+			{
+				ticker.add(this.update, this);
+			}
+			this._autoUpdate = !!value;
+		}
+	});
+
+	/**
 	 * Updates all particles spawned by this emitter and emits new ones.
 	 * @method update
 	 * @param {Number} delta Time elapsed since the previous frame, in __seconds__.
 	 */
 	p.update = function(delta)
 	{
+		if (this._autoUpdate)
+		{
+			delta *= 0.01;
```

Comment:
In pixi-animate [I set up some code](https://github.com/jiborobot/pixi-animate/blob/master/src/animate/MovieClip.js#L285) to convert from ticker delta (which is not hundredths of a second) to actual elapsed seconds. Would that be worth reusing here?